frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -1524,9 +1524,26 @@ class Worksheet extends React.Component {
         } else {
             var ws = _.clone(this.state.ws);
             for (var i = 0; i < partialUpdateItems.length; i++) {
-                if (!partialUpdateItems[i]) continue;
+                if (
+                    !partialUpdateItems[i] ||
+                    !(
```

Comment:
Can you add a comment explaining what this is looking for?

Reply:
Added!